pyproject.toml

```diff
@@ -101,8 +101,8 @@ multidict = "^4.6"
 aiohttp = "~3.6"
 questionary = "~1.5.1"
 prompt-toolkit = "^2.0"
-python-socketio = ">=4.4,<6"
-python-engineio = ">=3.11,<3.14"
+python-socketio = ">=5,<6"
```

Comment:
is there a specific reason we need to change the `python-socketio` version range?